service/src/test/java/greencity/service/EventCommentServiceImplTest.java

```diff
@@ -119,4 +118,47 @@ void getAllActiveCommentsEventNotFoundException() {
         when(eventRepo.findById(1L)).thenReturn(Optional.empty());
         assertThrows(NotFoundException.class, () -> eventCommentService.countComments(eventId));
     }
+
+    @Test
+    void delete() {
+        UserVO userVO = getUserVO();
+        Long commentId = 1L;
+
+        when(eventCommentRepo.findById(commentId))
+            .thenReturn(Optional.ofNullable(ModelUtils.getEventComment()));
+
+        eventCommentService.delete(commentId, userVO);
```

Comment:
Please add some verify to remove code smell

Reply:
Done